src/hotspot/cpu/aarch64/macroAssembler_aarch64.cpp

```diff
@@ -5528,9 +5528,9 @@ static int reg2offset_out(VMReg r) {
   return (r->reg2stack() + SharedRuntime::out_preserve_stack_slots()) * VMRegImpl::stack_slot_size;
 }
 
-// On 64 bit we will store integer like items to the stack as
-// 64 bits items (Aarch64 abi) even though java would only store
-// 32bits for a parameter. On 32bit it will simply be 32 bits
+// On64 bit we will store integer like items to the stack as
```

Comment:
Missing space.

Reply:
Oh, looks like i deleted the wrong space by accident.